packages/browser-sdk/src/providers/injected/index.ts

```diff
@@ -151,6 +155,31 @@ export class InjectedProvider implements Provider {
       throw new Error(`Invalid provider for injected connection: ${authOptions.provider}. Must be "injected"`);
     }
 
+    const requestedWalletId = authOptions.walletId || "phantom";
+    if (requestedWalletId === "phantom") {
+      // Default to Phantom injected provider
+      this.selectedWalletId = "phantom";
```

Comment:
Should we be assigning this before the `addressType` and registry checks below? We're otherwise setting what could be an "invalid" wallet ID.